.github/workflows/tests.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           && !contains(github.event.pull_request.labels.*.name, 'ci:run-full-test-suite')
           || false
          }}
-      pytest_store_durations: false
+      pytest_store_durations: true

```

Comment:
Since I already found out a while back that setting this to `true` doesn't have noticeable impact on CI runtime, I' thinking it makes sense to just keep this as `true` by default. That way we can create artifacts easily and update durations in any PR that creates major imbalances.